tests/manage/pv_services/pvc_resize/test_pvc_expansion_when_full.py

```diff
@@ -0,0 +1,265 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.resources.pod import get_used_space_on_mount_point
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version, ManageTest, tier2, polarion_id
+)
+from ocs_ci.utility.prometheus import PrometheusAPI, check_alert_list
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@skipif_ocs_version('<4.5')
+@polarion_id('OCS-301')
+class TestPvcExpansionWhenFull(ManageTest):
+    """
+    Tests to verify PVC expansion when the PVC is 100% utilized.
+    Verify utilization alert will stop firing after volume expansion.
+
+    """
+    @pytest.fixture(autouse=True)
+    def setup(self, create_pvcs_and_pods):
+        """
+        Create PVCs and pods
+
+        """
+        self.pvc_size = 4
+        self.pvcs, self.pods = create_pvcs_and_pods(
```

Comment:
with these params, only 1 pod is getting created right? I think 1 pod is enough

Reply:
It will create 1 cephfs PVC and one rbd PVC. 1 pvc will be attached to one pod. So total 2 pods will be created.